.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+(Delete these instructions before hitting **Submit**.)
```

Comment:
For both this and the PR template, why not a concise checklist with fill in the blanks and links back to the dev guides for more information?  I think a wall of text in the wrong approach and will intimidate people, I also doubt everyone will read the whole thing or delete it before submitting.